### DISCUSSION TITLE
How to add extra parameters in Document's Meta class.

### DISCUSSION THREAD

**Discussion by girase1pranay** [dymmond/mongoz#49] [Q&A]
I am trying to define parameters in the Meta class of Document with a straight forward way eg.
 `class Meta:
    custom_parameter = "This is a custom parameter"` 
but not able to access extra parameters with the Document class.
Can anyone please give a solution about how extra parameters can be added in the Meta class of Document?

**Comment by tarsil** [comment]
@girase1pranay because that Meta of a Document does not allow you to do that, naturally. It was not designed to allow custom parameters, only the ones that Mongoz uses and needs

**Reply by girase1pranay** [reply]
@tarsil Is there any workaround to add extra parameters in the Meta class?

**Reply by tarsil** [reply]
What's the purpose of the extra custom parameters? Why not creating an object yourself (pydantic base model for example) and simply add whatever there and use that object?

**Reply by tarsil** [reply]
@girase1pranay if you give a proper use case for this, we might be able to propose that and add it as a feature
